### DISCUSSION TITLE
Automation or cron to export App data every 6am (example time)

### DISCUSSION THREAD

**Discussion by sghriyadh** [Budibase/budibase#10614] [Help]
Hi

i am on self hosted Docker version on windows 10.

I would like to backup app data as automation or cron job at certain time daily.  is this possible from inside Budibase?

**Comment by melohagan** [comment]
Hey @sghriyadh 

You cannot backup your apps with a cron job, however every time your app is published a backup will automatically be created, if you have backups enabled: 
https://docs.budibase.com/docs/backups-1
